pages/linux/xtrlock.md

```diff
@@ -0,0 +1,15 @@
+# xtrlock
+
+> Lock the X display until the user supplies their password.
+
+- Lock the display and show a padlock instead of the cursor:
+
+`xtrlock`
+
+- Display a blank screen as well as the padlock cursor:
+
+`xtrlock -b`
+
+- Fork the xtrlock process and return immediately:
```

Comment:
What is this useful for ?

Reply:
For instance, you can do custom lock screen: `xtrlock -f && sm locked`